bin/runMethPipe.sh

```diff
@@ -0,0 +1,122 @@
+#/bin/bash
+#Extraction of the methylseq Nextflow pipeline (v 1.4.0)
+#Requires a pair of fastq files and a reference directory that
+#has been prepared with Bismark
+
+usage()
+{
+    echo "usage: runMethPipe  output-dir fastq1 fastq2 reference-dir"
+}
+
+if [ "$#" -ne 4 ]; then
+  echo
+  usage
+  echo
+  exit
+fi
+
+OUTD=$1
+FASTQ1=$2
+FASTQ2=$3
+REF=$4
+
+CPUs=16
```

Comment:
The caller must always be able to set the number of CPUs.  Make it possible to override `CPUs` by either:

1. Optional 5th ARG
2. Test for ENV var and fall back to 16 if not set.

From experience this is a potential source for future GitHub issues.